mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -168,8 +168,17 @@ func stop() error {
 			}
 		}
 	}
-
-	log.Info("Meshery is stopped.")
+	// Wait for meshery to stop
+	done, err := mesheryStopCheck()
+	if err != nil {
+		log.Info(err)
+	}
+	// If stop is not done until timeout, display info
+	if !done {
+		log.Info("Meshery has not stopped yet.\nPlease check the status of the pods by executing “mesheryctl system status” and Meshery-UI endpoint with “mesheryctl system dashboard”.")
```

Comment:
whats the use of checking the Meshery-UI endpoint in this stop workflow?
I feel, the below log would be sufficient:

"Meshery has not stopped yet.\nPlease check the status of the pods by executing “mesheryctl system status”".
Why I am not in favor of asking users to run "mesheryctl system dashboard" is - this command might fail as some resources might have got deleted already.
// @piyushsingariya

Reply:
Ah yes. I copied this message from the start workflow.